libraries/AP_HAL_ChibiOS/hwdef/BrahmaF4/hwdef.dat

```diff
@@ -0,0 +1,139 @@
+# hw definition file for processing by chibios_pins.py
+# for Brahma F405 FW
+#File Author shadow.walker.8642@gmail.com
+
+USB_STRING_MANUFACTURER "DM"
+USB_STRING_PRODUCT "BrahmaF4"
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 7701
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+STM32_ST_USE_TIMER 5
+
+# flash size
+FLASH_SIZE_KB 1024
+FLASH_RESERVE_START_KB 48
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+################# UART Configuration ##################
+SERIAL_ORDER OTG1 USART1 USART3 USART6 UART4 
+
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+PA10 USART1_RX USART1
+PA9 USART1_TX USART1
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_RCIN
+
+PB10 USART3_TX USART3 NODMA
+PB11 USART3_RX USART3 NODMA
+
```

Comment:
```suggestion
define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_ESCTelemetry
```
I meant the UART3 protocol should be set to ESC since its pinned out as such on the ESC connector